nixos/modules/services/security/bitwaren_rs/default.nix

```diff
@@ -0,0 +1,477 @@
+{ config, lib, pkgs, ... }:
+
+with import <nixpkgs/lib>;
```

Comment:
replace by `with lib;`

Reply:
Whoops, seemed not to have undone that test case, it was to make sure that it would work with the standard lib rather than my modified lib.